external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaDataConsumerSuite.scala

```diff
@@ -95,10 +100,12 @@ class KafkaDataConsumerSuite extends SharedSQLContext with PrivateMethodTester {
       try {
         val range = consumer.getAvailableOffsetRange()
         val rcvd = range.earliest until range.latest map { offset =>
-          val bytes = consumer.get(offset, Long.MaxValue, 10000, failOnDataLoss = false).value()
-          new String(bytes)
+          val record = consumer.get(offset, Long.MaxValue, 10000, failOnDataLoss = false)
+          val value = new String(record.value(), StandardCharsets.UTF_8)
+          val headers = record.headers().toArray.map(header => (header.key(), header.value())).toSeq
+          (value, headers)
         }
-        assert(rcvd == data)
+        data === rcvd
```

Comment:
Why remove the assertion?

Reply:
@srowen It was just a mistake. It is now updated with a deep comparison.